com.unity.cinemachine/Editor/Utility/EditorWithIcon.cs

```diff
@@ -0,0 +1,31 @@
+using Cinemachine;
+using Cinemachine.Editor;
+using UnityEditor;
+
+namespace Editor.Utility
```

Comment:
I have several issues with this.

- Wrong namespace.  Should be Cinemachine.Editor.
- It's public - anyone using it will get a CmCamera icon by default, whether it's appropriate or not
- It knows about our classes - a pattern to be avoided as it's not sustainable.  

I think you should get rid of this base class.